Backend/src/main/java/com/onbank/schedulders/TransferSessions.java

```diff
@@ -0,0 +1,29 @@
+package com.onbank.schedulders;
+
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TransferSessions {
+
+    @Scheduled(cron = "0 30 8 * * ?")
```

Comment:
Proponuję ustawienia CRONa trzemać w propertiesach i wstrzykiwać za pomocą @Value. Pozwoli nam to na zmiany godzin sesji bez konieczności zmian w kodzie. Dodatkowo zastanawiam się, czy nie dałoby się zamiast 4 cronów zrobić 2? Wydaje mi się że jest możliwość odpalenia crona 2 razy dziennie, możesz spróbować w ten sposób: 0 30 8,14 * * ? 
Oczywiście przetestuj może na sekundach czy coś, a nie na godzinach :D Nie jestem pewien że to zadziała, cron to umożliwia na pewno, ale nie wiem jak ten javowy.